docs/SPEC.md

```diff
@@ -159,7 +159,7 @@ Examples:
     2.71828
     .26
 
-[IMPL#254](https://github.com/influxdata/platform/issues/254) Parse float literals
+<!-- [IMPL#254](https://github.com/influxdata/platform/issues/254) Parse float literals -->
```

Comment:
I wonder if there's utility in keeping this commented out. Should we not just remove it?

I wonder if this, and the others are commented out because no equivalent issue was filed in the current repo(s).

Reply:
Ah yes, lemme go back and review these!